static/app/components/events/interfaces/crashContent/exception/sourceMapDebug.tsx

```diff
@@ -24,29 +23,26 @@ import {
   useSourceMapDebugQueries,
 } from './useSourceMapDebug';
 
-const platformDocsMap: Record<string, string> = {
-  javascript: 'javascript',
-  node: 'node',
-  'javascript-react': 'react',
-  'javascript-angular': 'angular',
-  // Sending angularjs to angular docs since it's not supported, has limited docs
-  'javascript-angularjs': 'angular',
-  // Sending backbone to javascript docs since it's not supported
-  'javascript-backbone': 'javascript',
-  'javascript-ember': 'ember',
-  'javascript-gatsby': 'gatsby',
-  'javascript-vue': 'vue',
-  'javascript-nextjs': 'nextjs',
-  'javascript-remix': 'remix',
-  'javascript-svelte': 'svelte',
+const sdkDocsMap: Record<string, string> = {
```

Comment:
you might need to update the allowed platforms to also check the sdk name? https://github.com/getsentry/sentry/blob/master/static/app/components/events/interfaces/crashContent/exception/useSourceMapDebug.tsx#L133